src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.html

```diff
@@ -13,43 +13,37 @@
 <section class="mat-typography" [class.small]="isSmall">
     <div class="button-examples-container">
         <h5>Click on the buttons below to see the spinners.</h5>
```

Comment:
This h5 is not rendered as our typography. You may want to use `mat-body-1` class plus a 32px bottom margin on a div.

Reply:
@emclaug2 @daileytj maybe we want to add native elements like h1, h2, ..., h6 to the ng theme? It is supported by Angular Material, but not in our pxblue themes.

Reply:
I'm surprised that doesn't work, but it looks like you are right —  seems we are only applying our style tweaks on the class names and not the tags: https://github.com/pxblue/themes/blob/dev/angular/_typography.scss

Reply:
Added  mat-body-1 class to h5 with 32px bottom margin.